### DISCUSSION TITLE
introduction and inquiry

### DISCUSSION THREAD

**Discussion by ToobaJamal** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#11] [Outreachy 2022] [answered]
Hello, @alswajiab and @BatoolMM!
Hope you are doing well. I am an outreachy contributor interested in this project. Since improving website accessibility is one of the preferred tasks that a successful candidate will perform, am I allowed to change the color template (if necessary) to improve the accessibility or keep the same styling?
Thanks!

**Comment by BatoolMM** [comment, marked as answer]
Hi @ToobaJamal, If you want to change colour template, please change it in a separate PR, different than the PR which you will be adding a subpage as outlined in the [Outreachy issue](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12). Let us know if you have any other questions!

**Reply by ToobaJamal** [reply]
@BatoolMM, the google docs table is already filled, can I insert another cells in it?
